src/python/pants/backend/python/lint/black/rules_integration_test.py

```diff
@@ -111,8 +111,6 @@ def test_passing(rule_runner: RuleRunner, major_minor_interpreter: str) -> None:
     )
     assert len(lint_results) == 1
     assert lint_results[0].exit_code == 0
-    assert "1 file would be left unchanged" in lint_results[0].stderr
-    assert "1 file left unchanged" in fmt_result.stderr
```

Comment:
Perhaps keep the asserts, but with a `not in` instead?

Reply:
Asserting in each test that _some_ output doesn't exist feels funny to me.
I think perhaps a new test is best for this.